### ISSUE TITLE
2.2 win7 x64 gif/video delete "The action can't be completed because the file is open in Telegram Desktop"

### ISSUE THREAD

**Issue by ghost** [telegramdesktop/tdesktop#7818]
After you send gif/video you can't delete it from your PC because "The action can't be completed because the file is open in Telegram Desktop", you have to close telegram every time, delete the file you just sent and reopen telegram.

**Comment by Aokromes** [comment]
there is no need of close one older ticket labeled as bug and open another simply to update version....

**Comment by ilya-fedin** [comment]
AFAIK, tdesktop doesn't redownload uploaded files and uses the local ones
I.e. the file that you can't delete is really used

**Comment by AgentFire** [comment]
You don't have to close Telegram btw, only switch chat tab to another chat. Switch it back - and the file doesn't get reopened too.

**Comment by AgentFire** [comment]
The issue still exists, why close it?

**Comment by imashxtyz** [comment]
update 2.5.1

**Comment by NullEnt1ty** [comment]
Issue still exists in 2.8.4

**Comment by AgentFire** [comment]
2.8.6 beta - still exists.

**Comment by MouamleH** [comment]
2.8.11 still exists

**Comment by NullEnt1ty** [comment]
Still happens with 3.5

**Comment by AgentFire** [comment]
3.5.1 still exists.

**Comment by NullEnt1ty** [comment]
Problem still exists.

**Comment by KieranDevvs** [comment]
Problem still exists in 4.2.4.

**Comment by ILoveSkilledCoders** [comment]
Still an issue in 4.3.3

My guess is they aren't going to fix it.

**Comment by ilya-fedin** [comment]
That's right, it was mentioned in this issue it's intended behavior. Please read old comments for details.

**Comment by KieranDevvs** [comment]
> That's right, it was mentioned in this issue it's intended behavior. Please read old comments for details.

I've just read back through all the comments, no where does anyone state that this is intended and it definitely should NOT be intended.

**Comment by ilya-fedin** [comment]
@KieranDevvs the second comment in the issue, authored by me

It could be stated as bad design, yeah, but it's not a bug as intended behavior, sorry

**Comment by AgentFire** [comment]
No interest is lost.

**Comment by soredake** [comment]
Not stale.
